src/ucp/core/ucp_worker.c

```diff
@@ -488,6 +496,8 @@ ucs_status_t ucp_worker_get_address(ucp_worker_h worker, ucp_address_t **address
             goto err_free;
         }
 
+        VALGRIND_CHECK_MEM_IS_DEFINED(rsc_addr, rsc_addr_length);
+
```

Comment:
@yosefe,
I'm not familiar with Valgrind enough, so can you explain how the line above works? As I was able to understand `VALGRIND_CHECK_MEM_IS_DEFINED` returns the pointer on the first undefined byte. But you don't check for it.
Maybe you meant `VALGRIND_MAKE_MEM_DEFINED` ?